File: src/main/java/org/example/bankramenserver/global/ai/GeminiProperties.java
package org.example.bankramenserver.global.ai;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "gemini")
public class GeminiProperties {

    private String apiKey;
    private String model = "gemini-2.5-flash";
    private String baseUrl = "https://generativelanguage.googleapis.com/v1beta";
}

File: src/main/kotlin/org/example/balogserver/global/ai/GeminiProperties.kt
package org.example.balogserver.global.ai

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "gemini")
class GeminiProperties { var apiKey: String? = null; var model: String = "gemini-2.5-flash"; var baseUrl: String = "https://generativelanguage.googleapis.com/v1beta" }
